open-machine-learning-jupyter-book/assignments/ml-fundamentals/study-the-solvers.md

```diff
@@ -0,0 +1,15 @@
+# Study the solvers
```

Comment:
Could you please double check if this page could be linked correctly from another page? Where you are creating a link pointing to this, please check if that URL is correct.

Reply:
yes, it is correct, I can open the link to this file. the link is in this file : "more-classifiers.md"